romancal/tweakreg/tweakreg_step.py

```diff
@@ -151,7 +151,7 @@ def process(self, input):
         # Build the catalogs for input images
         for i, image_model in enumerate(images):
             if hasattr(image_model.meta, "tweakreg_catalog"):
-                catalog = Table.read(image_model.meta.tweakreg_catalog)
+                catalog = Table.read(image_model.meta.tweakreg_catalog, format="ascii")
```

Comment:
Is it necessary to restrict format to ascii?

Reply:
For now, yes.
The new implementation of Source Detection will append the catalog data (as `np.array`) to that attribute, allowing us to drop the format constraint. The ticket for that work can be found [here](https://jira.stsci.edu/browse/RCAL-531).